.gitignore

```diff
@@ -15,3 +15,4 @@ Gemfile.lock
 node_modules/
 package-lock.json
 coverage/
+public/
```

Comment:
This looks unrelated

Reply:
It's still needed, so split it to a separate PR.